shopfloor_workstation_label_printer/models/res_users.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    default_label_printer_id = fields.Many2one(
+        comodel_name="printing.printer", string="Default Label Printer"
+    )
+
+    @api.model
+    def _register_hook(self):
+        super()._register_hook()
+        self.SELF_WRITEABLE_FIELDS.extend(["default_label_printer_id"])
+        self.SELF_READABLE_FIELDS.extend(["default_label_printer_id"])
+
+    def get_delivery_label_printer(self, delivery_type):
```

Comment:
who's using this method? :thinking:

Reply:
Well this module started with a mechanism for a 2nd printer configured for a specific delivery. But the requirements changed and that improvement was not needed any more.
This method was useful for that use case and I left it there.
It is used by the backend when printing.

Reply:
I cannot find it... where?

Reply:
In a specific module that handle the printing